components/omega/src/ocn/auxiliaryVars/TracerAuxVars.h

```diff
@@ -0,0 +1,101 @@
+#ifndef OMEGA_AUX_TRACER_H
+#define OMEGA_AUX_TRACER_H
+
+#include "DataTypes.h"
+#include "HorzMesh.h"
+#include "OmegaKokkos.h"
+#include "auxiliaryVars/LayerThicknessAuxVars.h"
+
+#include <string>
+
+namespace OMEGA {
+
+class TracerAuxVars {
+ public:
+   Array3DReal HTracersOnEdge;
+   Array3DReal Del2TracersOnCell;
```

Comment:
Do we need to have methods for registering these variables with IOStreams, like we have for all other auxiliary variables ?